### PR TITLE
Restart Game

### PR DESCRIPTION
**Pull request by mentor-tara**
### Approve and merge this PR to start a new game.

This is the new board:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   |   |   |   |   |   |
| **2** |   |   |   |   |   |   |   |   |
| **3** |   |   |   |   |   |   |   |   |
| **4** |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |
| **6** |   |   |   |   |   |   |   |   |
| **7** |   |   |   |   |   |   |   |   |
| **8** |   |   |   |   |   |   |   |   |

[](restart-2)

### CODE REVIEW

File: commands.json [hunk 1]
```diff
@@ -1,12 +1,3 @@
...
-    "clear A8",
-    "clear A1",
-    "flag A5",
-    "flag A6",
-    "clear A4",
-    "clear B4",
-    "clear C4"
-  ]
-}
+  "commands": []
```
Comment (min-coding) [inline commands.json:2]: good start

File: internal/game [hunk 2]
```diff
@@ -1 +1 @@
...
...LCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjcsInlwb3MiOjYsInZhbHVlIjowLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX1dLFt7Inhwb3MiOjAsInlwb3MiOjcsInZhbHVlIjoyLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjoxLCJ5cG9zIjo3LCJ2YWx1ZSI6Ijpib21iOiIsImlzTWluZSI6dHJ1ZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6MiwieXBvcyI6NywidmFsdWUiOiI6Ym9tYjoiLCJpc01pbmUiOnRydWUsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjMsInlwb3MiOjcsInZhbHVlIjoiOmJvbWI6IiwiaXNNaW5lIjp0cnVlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjo0LCJ5cG9zIjo3LCJ2YWx1ZSI6MSwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9LHsieHBvcyI6NSwieXBvcyI6NywidmFsdWUiOjAsImlzTWluZSI6ZmFsc2UsImlzUmV2ZWFsZWQiOmZhbHNlLCJpc0ZsYWdnZWQiOmZhbHNlfSx7Inhwb3MiOjYsInlwb3MiOjcsInZhbHVlIjowLCJpc01pbmUiOmZhbHNlLCJpc1JldmVhbGVkIjpmYWxzZSwiaXNGbGFnZ2VkIjpmYWxzZX0seyJ4cG9zIjo3LCJ5cG9zIjo3LCJ2YWx1ZSI6MCwiaXNNaW5lIjpmYWxzZSwiaXNSZXZlYWxlZCI6ZmFsc2UsImlzRmxhZ2dlZCI6ZmFsc2V9XV0sImdhbWVDb3VudCI6MywibWluZXNGb3VuZCI6MCwiZmFsc2VNaW5lcyI6MCwic3RhdHVzIjoiT05HT0lORyIsIm1vdmVzTWFkZSI6MCwib3B0aW9ucyI6eyJyb3dzIjo4LCJjb2xzIjo4LCJtaW5lcyI6MTB9fQ==
```
Comment (min-coding) [inline internal/game:1]: something new

**Review by min-coding — APPROVED** [review]
thanks

**Comment by min-coding** [comment]
restart game by merging into main

File: commands.json (hunk 1 above)
Comment (dlogpit-SRE) [inline commands.json:2]: check this out

**Review by dlogpit-SRE — COMMENTED** [review]
Nice

**Pull request merged**